docs/newbs_learn_more_resources.md

```diff
@@ -1,13 +1,15 @@
 # Learning Resources
```

Comment:
this page needs to be added to both _summary.md and _sidebar.md, so that it shows up in the sidebar.

Reply:
What's the difference between `_summary.md` and `_sidebar.md`? I just looked at them and the files are identical.

Reply:
Not sure, honestly.  Other than maybe where they're used

Reply:
I looked it up, and Docsify uses `_sidebar.md`. `_summary.md` seems to be a vestige of when our docs ran on GitBook.

Reply:
Okay.  Then really, we should probably remove one of these?  @skullydazed 
That or keep and maintain both, for compatibility?

Reply:
In absence of review from skully, I'm just going to keep both files until I'm told to do otherwise.